### ISSUE TITLE
Hello author, I tried your latest ver XPS15-9550-OC-INTEL-201218 but it panic

### ISSUE THREAD

**Issue by tiendatnguyen997** [xxxzc/xps15-9550-macos#85]
Hello, 
I tried your latest version of XPS15-9550-OC-INTEL-201218, but when I boot, my laptop ( 5510, i7 6820HQ, 16Gb Ram, SSD 512 SM961, 4K screen) had a panic. This is ** In Memory Panic Stackshot Succeeded ** Bytes Traced 3721
Attemting system restart...
MACH Reboot
![image](https://user-images.githubusercontent.com/74667446/102619191-a11a5e80-416e-11eb-8e72-10ce509779f0.png)
![image](https://user-images.githubusercontent.com/74667446/102619206-a8416c80-416e-11eb-98c3-e17471ab0b42.png)
Can you help me, please!
Thank you so much.

**Comment by OscarWoHA** [comment]
This issue has been reported previously in #81. Please follow along there and close this issue.

**Comment by tiendatnguyen997** [comment]
> This issue has been reported previously in #81. Please follow along there and close this issue.

Thank you so much for reply. I read this solution, but I did not understand how to gat the CR2 code and where to add it?
In my screen error did not have the line about CR2
![2020-12-19 (1)](https://user-images.githubusercontent.com/74667446/102689996-13557680-4235-11eb-84a7-5153f89e27ef.png)

**Comment by xxxzc** [comment]
@OscarWoHA have you fixed the issue you posted?

**Comment by xxxzc** [comment]
@tiendatnguyen997
1. can you change `DevirtualiseMmio` to false and see what will happen?
2. set both `DevirtualiseMmio` and `RebuildAppleMemoryMap` to  true

Changing `Target` to `83`(0x53) to enable saving verbose log to EFI or following this debug guide: https://dortania.github.io/OpenCore-Install-Guide/troubleshooting/debug.html#opencore-debugging

**Comment by OscarWoHA** [comment]
@xxxzc No, but since I'm having the same issue as this dude I thought I'd close my issue.

**Comment by tiendatnguyen997** [comment]
> @tiendatnguyen997
> 
> 1. can you change `DevirtualiseMmio` to false and see what will happen?
> 2. set both `DevirtualiseMmio` and `RebuildAppleMemoryMap` to  true
> 
> Changing `Target` to `83`(0x53) to enable saving verbose log to EFI or following this debug guide: https://dortania.github.io/OpenCore-Install-Guide/troubleshooting/debug.html#opencore-debugging

I did that, but it stayed at here 
![20201222_210506](https://user-images.githubusercontent.com/74667446/102897312-62501580-449a-11eb-99f6-adc094e52011.jpg)

**Comment by xxxzc** [comment]
Please upload your EFI

**Comment by tiendatnguyen997** [comment]
> Please upload your EFI

This is my EFI. Thank you so much
https://drive.google.com/drive/folders/1LFx1C_MfX4HkdEwmCufMwHbNqXuTeOzl?usp=sharing

**Comment by xxxzc** [comment]
could you try https://github.com/wmchris/DellXPS15-9550-OSX

**Comment by imanng** [comment]
@tiendatnguyen997  you can try new EFI, it works
